Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
```

Comment:
should i remove the whole gem file and how should i remove that @SidharthBansal

Reply:
i am unable to understand that ...please help me in this @SidharthBansal  and also explain this (Please remove yarn.lock and public/lib/emoji.js also from this commit.
Please link to the issue in the pr body so that we can check your work.)

Reply:
> should i remove the whole gem file and how should i remove that @SidharthBansal
Yes ! Just delete Gemfile.lock before making a commit, do NOT delete the Gemfile. Similarly for others.